HW02_B_Mimic_starter.Rmd

```diff
@@ -91,7 +128,15 @@ The above graph lets you see some colobrlind friendly palettes. For the graph be
 Now using the above mpg dataset, make this graph
 
 ```{r graph 4 code}
-
+ggplot(mpg, aes(class,cty))+
+  scale_fill_brewer(palette = "Set2")+
+  scale_color_brewer(palette = "Set2")+
+  geom_boxplot(outlier.alpha=0, alpha=0)+
+  coord_flip()+
+  geom_point(aes(color=class), position= position_jitter(height=0))+

```

Comment:
I didn't add height = 0 and I got randomly arranged dots instead of aligned ones. Thanks because now I know how to fix it, but I am still confused about what did setting height to 0 actually change...?

Reply:
the position_jitter (height=) sets the resolution of the jitters on the y axis; the default is set to 0.4. Here it is actually x-axis because I flipped the coordinate. Since the x axis (Car class) is a categorical variable that composed integers. So in order to have the jitter only be set to integers, height should be set to 0. I didn't find much about this function, so this is just what I think it might be:)

Reply:
For the dots you got by not setting height=0, I'm guessing its arranged by every 0.4?